benchmark/es/obj-bench.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const common = require('../common.js');
+const Obj1 = require('../fixtures/moduleA');
+const Obj2 = require('../fixtures/moduleB');
+
+const bench = common.createBenchmark(main, {
+  // TODO(jasnell): A and B are not great labels, not sure what would be better
+  method: ['A', 'B'],
+  access: ['direct', 'indirect'],
+  millions: [10]
+});
+
+function run(n, obj, direct) {
+  var i;
+  if (direct) {
+    bench.start();
+    for (i = 0; i < n; i++) {
+      obj.A();
+      obj.B();
+      obj.C;
+    }
+    bench.end(n / 1e6);
+  } else {
+    const A = obj.A;
+    const B = obj.B;
+    bench.start();
+    for (i = 0; i < n; i++) {
+      A();
+      B();
+      obj.C;
```

Comment:
Which specifically? The `obj.C` bit? Basically checking any accumulative effect. It may not make sense to keep tho :-)